<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meter_page_</class>
 <widget class="QWidget" name="meter_page_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Meter Page</string>
  </property>
  <widget class="QGroupBox" name="serial_port_gb">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>2</y>
     <width>272</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Serial Port Info</string>
   </property>
   <widget class="QLabel" name="port_lbl">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>22</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="baudrate_lbl">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>62</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="databits_lbl">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>102</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data Bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="parity_lbl">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>142</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopbits_lbl">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>182</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="parity_cbb">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="databits_cbb">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="baudrate_cbb">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="stopbits_cbb">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="port_cbb">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>20</y>
      <width>220</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="connectstatus_txta">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>258</y>
      <width>260</width>
      <height>185</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="testconnect_btn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>260</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="receive_gb">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>537</y>
     <width>891</width>
     <height>85</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Receive</string>
   </property>
   <widget class="QPlainTextEdit" name="receive_txta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>870</width>
      <height>60</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="send_cmd_gb">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>454</y>
     <width>891</width>
     <height>85</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Send CMD</string>
   </property>
   <widget class="QPlainTextEdit" name="sendcmd_txta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>870</width>
      <height>60</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="meter_result_gb">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>2</y>
     <width>491</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Isolight Puck Color Light Meter</string>
   </property>
   <widget class="QLabel" name="grcct_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GRCCT</string>
    </property>
   </widget>
   <widget class="QLabel" name="grl_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GRL</string>
    </property>
   </widget>
   <widget class="QLabel" name="grxyz_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GRXYZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="gryxy_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GRYXY</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="grcct_txt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="grl_txt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="grxyz_x_txt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="grxyz_y_txt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="grxyz_z_txt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="gryxy_y1_txt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="gryxy_y2_txt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="gryxy_x_txt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="y2_lbl">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>170</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x2_lbl">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>170</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="y3_lbl">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>170</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x1_lbl">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>120</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="y1_lbl">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>120</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="z1_lbl">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>120</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sendcmd_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send CMD</string>
    </property>
   </widget>
   <widget class="QLabel" name="cmd_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CMD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sendcmd_cbb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="threads_gb">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>2</y>
     <width>121</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>Threads</string>
   </property>
   <widget class="QPushButton" name="play_btn">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>108</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>220</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_btn">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>164</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="intervaltime_ldn">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>60</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="intervaltime_lbl">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>22</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Interval Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="threadsstatus_txta">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>258</y>
      <width>110</width>
      <height>185</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>250</green>
          <blue>146</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>249</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>receive_gb</zorder>
  <zorder>send_cmd_gb</zorder>
  <zorder>meter_result_gb</zorder>
  <zorder>serial_port_gb</zorder>
  <zorder>threads_gb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
